.github/workflows/vast.yaml

```diff
@@ -1079,23 +607,448 @@ jobs:
+      - vast
+      - configure
+    if: ${{ always() && needs.configure.outputs.run-vast-plugins == 'true' }}
+    runs-on: ${{ matrix.plugin.os }}
+    container: ${{ matrix.plugin.container }}
+    strategy:
+      fail-fast: false
+      matrix:
+        plugin:
+          - name: Example Analyzer Debian
+            target: example-analyzer
+            path: examples/plugins/analyzer
+            dependencies:
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Example Analyzer macOS
+            target: example-analyzer
+            path: examples/plugins/analyzer
+            dependencies:
+            os: macos-latest
+            container:
+          - name: Example Pipeline Operator Debian
+            target: example-pipeline-operator
+            path: examples/plugins/pipeline_operator
+            dependencies:
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Example Pipeline Operator macOS
+            target: example-pipeline-operator
+            path: examples/plugins/pipeline_operator
+            dependencies:
+            os: macos-latest
+            container:
+          - name: Broker Debian
+            target: broker
+            path: plugins/broker
+            dependencies:
+              - libbroker-dev
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Sigma Debian
+            target: sigma
+            path: plugins/sigma
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Sigma macOS
+            target: sigma
+            path: plugins/sigma
+            os: macos-latest
+            container:
+          - name: Parquet Debian
+            target: parquet
+            path: plugins/parquet
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Parquet macOS
+            target: parquet
+            path: plugins/parquet
+            os: macos-latest
+            container:
+          - name: Web Debian
+            target: web
+            path: plugins/web
+            dependencies:
+              - libasio-dev
+              - libhttp-parser-dev
+            os: ubuntu-20.04
+            container: debian:bullseye-slim
+          - name: Web macOS
+            target: web
+            path: plugins/web
+            dependencies:
+              - asio
+              - http-parser
+            os: macos-latest
+    env:
+      INSTALL_DIR: "${{ github.workspace }}/_install"
+      BUILD_DIR: "${{ github.workspace }}/_build"
+      CMAKE_GENERATOR: Ninja
+      CMAKE_MAKE_PROGRAM: ninja
+      CTEST_OUTPUT_ON_FAILURE: YES
+      DEBIAN_FRONTEND: noninteractive
+      HOMEBREW_GITHUB_API_TOKEN: ${{ github.token }}
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_INSTALL_CLEANUP: 1
+      HOMEBREW_NO_AUTO_UPDATE: 1
+      DESTDIR: "${{ github.workspace }}"
+    name: ${{ matrix.plugin.name }} Plugin
```

Comment:
This now renders as `$plugin $os Plugin`, which reads a bit weird. In order to align with the other steps, can we make this `VAST Plugins ($plugin, $os)`?

Reply:
done